setup.py

```diff
@@ -32,7 +32,7 @@ def read(filename):
     "pinject == 0.14.1",
     "six == 1.12.0",
     "dnspython == 1.16.0",
-    "k8s == 0.15.0",
+    "k8s == 0.17.0",
```

Comment:
Also as mentioned, it might be cleaner to open a new PR that just updates the library version and fixes the existing tests, so it's totally clear the new functionality doesn't impact anything (without it being enabled)

Reply:
It is done. check https://github.com/fiaas/fiaas-deploy-daemon/pull/128, the current PR was rebased on the top of that PR.